test/LidoStakedEthRewardsIntegration.t.sol

```diff
@@ -0,0 +1,116 @@
+    address operator = 0x8076807464DaC94Ac8Aa1f7aF31b58F73bD88A27;  // Operator multi-sig
+
+    address WETH   = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address WSTETH = 0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0;
+
+    uint256 REWARD_AMOUNT = 20 ether;
+    uint256 DURATION      = 30 days;
+
+    PullRewardsTransferStrategy transferStrategy;
+
+    address whale = 0xf8dE75c7B95edB6f1E639751318f117663021Cf0;  // 7-siblings wallet
+
+    function setUp() public {
+        vm.createSelectFork(getChain("mainnet").rpcUrl, 18_707_715);  // Dec 3rd, 2023
+
+        transferStrategy = new PullRewardsTransferStrategy(
+            address(incentives),
+            admin,
+            operator
+        );
+
+        deal(WSTETH, operator, REWARD_AMOUNT);
+
+        vm.prank(operator);
+        IERC20(WSTETH).approve(address(transferStrategy), REWARD_AMOUNT);
+
+        vm.prank(admin);
+        emissionManager.setEmissionAdmin(WSTETH, operator);
+    }
+
+    function _setupDistribution() internal {
+        RewardsDataTypes.RewardsConfigInput[] memory configs = new RewardsDataTypes.RewardsConfigInput[](1);
+
+        configs[0] = RewardsDataTypes.RewardsConfigInput({
+            emissionPerSecond: uint88(REWARD_AMOUNT / DURATION),
+            totalSupply:       0,  // Set by the rewards controller
+            distributionEnd:   uint32(block.timestamp + DURATION),
+            asset:             _getAToken(WETH),  // Rewards on WETH supplies
+            reward:            WSTETH,
+            transferStrategy:  transferStrategy,
+            rewardOracle:      IEACAggregatorProxy(aaveOracle.getSourceOfAsset(WSTETH))
+        });
+
+        vm.prank(operator);
+        emissionManager.configureAssets(configs);
+    }
+
+    function test_setup_distribution() public {
+        _setupDistribution();
+    }
+
+    function test_user_claim() public {
+        address claimAddress = makeAddr("claimAddress");
+        address[] memory assets = new address[](1);
+        assets[0] = _getAToken(WETH);
+
+        vm.prank(whale);
+        incentives.claimAllRewards(assets, claimAddress);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), 0);
+
+        _setupDistribution();
+
+        vm.prank(whale);
+        incentives.claimAllRewards(assets, claimAddress);
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), 0);
+
+        uint256 skipAmount = DURATION / 2;  // 50% of rewards distrubuted
+        skip(skipAmount);
+
+        // 7-siblings wallet should get about half of the rewards at this time
+        // 79k ETH deposit out of 157k total supplied
+        vm.prank(whale);
+        incentives.claimAllRewards(assets, claimAddress);
+        uint256 whaleReward1 = 5.028734135612479150 ether;
+        assertEq(IERC20(WSTETH).balanceOf(claimAddress), whaleReward1);
+        assertEq(IERC20(WSTETH).balanceOf(operator),     REWARD_AMOUNT - whaleReward1);
+
+        skip(DURATION - skipAmount);
```

Comment:
This is the same as `skipAmount`, we should just to `skip(skipAmount)` and add an inline comment about going to the end of DURATION

Reply:
Not necessarily. There could be a rounding error.

Reply:
With a duration set as constant to 30 days, I don't think it's a probem.

Reply:
If we want to keep it as is though, I'd work on the names. Sth like
```
uint256 firstSkipAmount = FULL_DURATION / 2;
skip(firstSkipAmount);
...
skip(FULL_DURATION - firstSkipAmount);

Reply:
What's the issue with the way it is? Say we want to change the duration to an odd number. The code above is more robust.

Reply:
I only advocate for a slight variable naming enhancement here.